corehq/apps/export/static/export/js/download_export.js

```diff
@@ -160,19 +156,19 @@ hqDefine('export/js/download_export', function () {
                         self.downloadInProgress(true);
                         self.progressModel.startMultimediaDownload(data.download_id);
                     } else {
-                        self._handlePrepareError(data);
+                        self.handleError(data);
                     }
                 },
-                error: self._handlePrepareError,
+                error: self.handleError,
             });
         };
 
-        self._handlePrepareError = function (data) {
+        self.handleError = function (data) {
             if (data && data.error) {
                 // The server returned an error message.
                 self.prepareExportError(data.error);
             } else {
-                self.prepareExportError("default");
+                self.prepareExportError(gettext("Sorry, there was a problem reaching the server. Please try again."));
```

Comment:
This is a bit off-topic, but do you know if we have any sort of convention for reusing these sorts of messages?  I imagine every UI has some subtle variation of the same few messages, each of which needs to be translated independently.

Reply:
I don't think we do, but it's a good idea and probably not difficult to implement: I'd imagine a module in `hqwebapp` just full of string constants that we'd use as error messages.